tests/utils/test_get_secrets.py

```diff
@@ -23,6 +24,10 @@ def reverser(key):
 
 
 def test_get_secret_from_env():
+    # Set the prefix to "MLRUN_K8S_SECRET__" as this test validates behavior with a defined prefix,
+    # while the default configuration has been changed to empty.
+    config.secret_stores.kubernetes.env_variable_prefix = "MLRUN_K8S_SECRET__"
+
```

Comment:
use a test fixture so it would revert the config setting after this specific test